app/services/users.js

```diff
@@ -16,8 +16,7 @@ exports.createUser = async payload => {
 
 exports.getUsers = async (page, limit) => {
   try {
-    let offset = 0;
-    offset += (page - 1) * limit;
+    const offset = 0 + (page-1) * limit; // eslint-disable-line
```

Comment:
This is fine, but it's better to disable the specific problematic rule than to disable all `eslint` rules, like this:
```
  // eslint-disable-next-line no-mixed-operators
  const offset = 0 + (page - 1) * limit;
```